modules/data-entry/src/main/frontend/src/questionnaire/FileQuestion.jsx

```diff
@@ -292,7 +292,11 @@ function FileQuestion(props) {
     return (
       <div>
         <Link href={fixFileURL(hrefs[idx], label)} target="_blank" rel="noopener" download underline="hover">{label}</Link>
-        { previewRenderer && previewRenderer(fixFileURL(hrefs[idx], label), label, idx) }
+        { previewRenderer?.(
+          fixFileURL(hrefs[idx], label),
+          label,
+          idx
+        ) }
```

Comment:
I find this kind of indentation easier to read:

```suggestion
        { previewRenderer?.(
            fixFileURL(hrefs[idx], label),
            label,
            idx
        )}
```